fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBStoreTimer.java

```diff
@@ -224,6 +238,76 @@ public String logKey() {
         }
     }
 
+    /**
+     * An aggregate over other count events.
+     */
+    public enum EventAggregates implements Aggregate, Event {
+        COMMITS("commits",
+                Events.COMMIT,
+                Events.COMMIT_FAILURE),
+        READS("reads",
+                Events.LOAD_META_DATA,
+                Events.LOAD_RECORD_STORE_STATE,
+                Events.LOAD_RECORD_STORE_INFO,
+                Events.LOAD_RECORD_STORE_INDEX_META_DATA,
+                Events.LOAD_RECORD,
+                Events.SCAN_RECORDS,
+                Events.SCAN_INDEX_KEYS,
+                Events.DIRECTORY_READ,
+                Events.SCOPED_DIRECTORY_LAYER_READ,
+                Events.EXTENDED_DIRECTORY_LAYER_READ,
+                Events.INTERNING_LAYER_READ,
+                Events.LOAD_BOUNDARY_KEYS,
+                Events.READ_SAMPLE_KEY),
+        WRITES("writes",
+                Events.SAVE_META_DATA,
+                Events.LOAD_RECORD_STORE_INFO,
+                Events.SAVE_RECORD,
+                Events.SAVE_INDEX_ENTRY,
```

Comment:
The aggregated time value for some of these will double count things that run in parallel or nested within one another. I'm not sure how easy it is to avoid being thrown off by that.

Reply:
I have removed the READ/WRITE/DELETE aggregates since, I agree, they can be confusing.

Reply:
Yeah, that may be simplest.